css/style.scss

```diff
@@ -0,0 +1,382 @@
+      align-content: center;
+      margin-right: 20px;
+      
+      img {
+            align-self: center;
+      }
+}
+.services-blocks__block__content {
+      width: 220px;
+
+      h3 {
+            font-family: lato;
+            color: $darkTextColor;
+            font-weight: bold;
+            font-size: 18px;
+            line-height: 18px;
+            margin-bottom: 7px;
+      }
+      p {
+            font-family: lato;
+            font-weight: normal;
+            font-style: normal;
+            color: $darkTextColor;
+            font-weight: 550;
+            font-size: 12px;
+            line-height: 22px;
+      }
+}
+
+                                          // PORTFOLIO
+
+.main-portfolio {
+      background-color: $mainBgColor;
+      border-bottom: 6px solid rgb(50, 55, 70);
+      display: flex;
+      justify-content: center;
+}
+.main-portfolio__article {
+      width: 1020px;
+      padding: 59px 40px 68px 40px;
+      box-sizing: border-box;
+}
+.main-portfolio__article__text-block {
+      font-family: lato;
+
+      h2 {
+            color: $darkTextColor;
+            font-weight: 900;
+            font-size: 30px;
+            line-height: 36px;
+            margin-bottom: 18px;
+      }
+      p {
+            color: $darkTextColor;
+            font-weight: lighter;
+            font-size: 18px;
+            line-height: 30px;
+      }
+}
+.button {
+      font-family: Lato;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 12px;
+      color: $darkTextColor;
+      text-align: center;
+      border: 1px solid $darkTextColor;
+      border-radius: 5px;
+      background-color: rgba($color: #000000, $alpha: 0);
+      outline: none;
+}
+.button:hover {
+      border: 1px solid $darkTextColorHover;
+}
+.button:active {
+      background-color: $darkTextColorHover;
+}
+.button:focus {
+      background-color: $darkTextColorHover;
+}
+.main-portfolio__article__buttons-block {
+      width: 290px;
+      height: 22px;
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      font-family: Lato;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 12px;
+      line-height: 12px;
+      margin-top: 20px;
+
+      .buttons-block__button__all {
+            width: 28px;
+      }
+      .buttons-block__button__webd {
+            width: 79px;
+      }
+      .buttons-block__button__graphicd {
+            width: 95px;
+      }
+      .buttons-block__button__artwork {
+            width: 58px;
+      }
+}
+
+.main-portfolio__article__gallery-block {
+      height: 621px;
+      width: 100%;
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      align-items: space-between;
+      margin-top: 18px;
+      overflow: hidden;
+
+      img {
+            margin-bottom: 20px;
+      }
+}
+
+                                          // FOOTER
+footer {
+      height: 76px;

```

Comment:
Please put this value to constant with sensible name.

Reply:
fixed